Automattic/wp-calypso
# Fetch `anchor_podcast` blog option

#### Changes proposed in this Pull Request

This PR adds the `anchor_podcast` option to the list of options that are retrieved from the site API endpoints. It'll be useful for determining if certain logic needs to run based on whether a site is already or not a podcasting site.

#### Testing instructions

- Checkout this branch and start Calypso locally.
- Apply D53313-code (counterpart diff of https://github.com/Automattic/jetpack/pull/17900) to your WP.com sandbox.
- Modify your `/etc/hosts` file and point `public-api.wordpress.com` to the IP of your WP.com sandbox.
- Go to http://calypso.localhost:3000/home.
- Pick a site where the `anchor_podcast` option is still unset.
- Open the JS console and check the value of the `anchor_podcast` option in the selected site state: `state.sites.items[state.ui.selectedSiteId].options.anchor_podcast`.
- Make sure it has a `false` value:
<img width="457" alt="Screen Shot 2020-11-27 at 13 04 15" src="https://user-images.githubusercontent.com/1233880/100447643-3f0b9380-30b1-11eb-8dc2-8fe48d81dd36.png">

- Set a `anchor_podcast` option value for your site:
  - Open a WP shell in your WP.com sandbox: `wp shell --user=<YOUR_WPCOM_USERNAME> --url=<YOUR_SITE_ADDRESS>`.
  - Update the blog option: `update_blog_option( <SITE_ID>, 'anchor_podcast', '3e51070' );`.
- Reload My Home.
- Check the value of the `anchor_podcast` option in the selected site state again.
- Make sure it has the value you entered above:
<img width="453" alt="Screen Shot 2020-11-27 at 13 06 40" src="https://user-images.githubusercontent.com/1233880/100447732-6a8e7e00-30b1-11eb-9670-30536e473670.png">

- Clean up the `anchor_podcast` option value from your site:
  - Open a WP shell in your WP.com sandbox: `wp shell --user=<YOUR_WPCOM_USERNAME> --url=<YOUR_SITE_ADDRESS>`.
  - Remove the blog option: `delete_blog_option( <SITE_ID>, 'anchor_podcast' );`.
